src/renderer/components/ft-shaka-video-player/player-components/FullWindowButton.js

```diff
@@ -64,6 +65,12 @@ export class FullWindowButton extends shaka.ui.Element {
       this.updateLocalisedStrings_()
     })
 
+    if (startInFullWindow) {
+      events.dispatchEvent(new CustomEvent('setFullWindow', {
+        detail: true
+      }))
+    }
```

Comment:
I might be missing something but what was the reason for doing this inside of the UI component instead of just doing it in the player component?

Reply:
This was my way of making sure that this event listener would fire properly. I can change this if there's a more elegant way to do it:

https://github.com/FreeTubeApp/FreeTube/blob/8c3311380bca5a9fda628130a723afbd3145a7a7/src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js#L1665-L1677